### ISSUE TITLE
build failed on windows 10

### ISSUE THREAD

**Issue by dirmansebastian** [paritytech/parity-bridge#186]
cargo:cargo 1.27.0 (1e95190e5 2018-05-27)
rustc:rustc 1.27.2 (58cc626de 2018-07-18)
solc:0.4.24+commit.e67f0147.Emscripten.clang

when i try to build i get the following error:

```
error: failed to run custom build command for `bridge v0.4.0 (file:///C:/Users/SebastianDirman/0keyethereum/parity-bridge/bridge)`
process didn't exit successfully: `C:\Users\SebastianDirman\0keyethereum\parity-bridge\target\release\build\bridge-0194787f66436b11\build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-changed=../contracts/bridge.sol
cargo:rustc-env=GIT_HASH=53c56a2440493323ea2a5ad122036e226d7aa610

--- stderr
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "The system cannot find the file specified." }', libcore\result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
warning: build failed, waiting for other jobs to finish...
error: build failed
```

**Comment by dirmansebastian** [comment]
i got these files in parity-bridge\target\release\build\bridge-0194787f66436b11:
![image](https://user-images.githubusercontent.com/15945292/43522299-a0cde40e-95a1-11e8-84e2-ad06adb2e49d.png)

**Comment by JosefJ** [comment]
Seems to be the same reason as in #187 
try to install alternatives to these: 
```
pkg-config
libssl-dev
```

**Comment by dirmansebastian** [comment]
damn, i still get same error : 

cargo build -p parity-bridge --release
   Compiling iovec v0.1.2
   Compiling bridge v0.4.0 (file:///C:/Users/SebastianDirman/0keyethereum/parity                  -bridge/bridge)
   Compiling kernel32-sys v0.2.2
   Compiling ws2_32-sys v0.2.1
error: failed to run custom build command for `bridge v0.4.0 (file:///C:/Users/S                  ebastianDirman/0keyethereum/parity-bridge/bridge)`
process didn't exit successfully: `C:\Users\SebastianDirman\0keyethereum\parity-                  bridge\target\release\build\bridge-08ee787f1372f559\build-script-build` (exit co                  de: 101)
--- stdout
cargo:rerun-if-changed=../contracts/bridge.sol
cargo:rustc-env=GIT_HASH=53c56a2440493323ea2a5ad122036e226d7aa610


--- stderr
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { cod                  e: 2, kind: NotFound, message: "The system cannot find the file specified." }',                   libcore\result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

warning: build failed, waiting for other jobs to finish...
error: build failed

**Comment by dirmansebastian** [comment]
fixed on ubuntu #187
